portfolio/portfolio.go

```diff
@@ -444,3 +444,39 @@ func StartPortfolioWatcher() {
 func GetPortfolio() *Base {
 	return &Portfolio
 }
+
+// ExchangeSupported checks if exchange is supported by portfolio address
+func ExchangeSupported(exchange, address string) (ret bool) {
```

Comment:
Please add test coverage for these funcs

Reply:
:D Yeah I added them in a commit this morning they are coming shortly